apps/ejabberd/src/mongoose_metrics.erl

```diff
@@ -276,9 +276,16 @@ create_metrics(Host) ->
     lists:foreach(fun(Name) -> ensure_metric(Name, counter) end,
                   get_total_counters(Host)).
 
-ensure_metric(Metric, Type) when is_list(Metric) ->
+ensure_metric(Metric, Type) when is_tuple(Type) ->
```

Comment:
I'm not sure the length is always the same, what I observed was 6 elements but I don't know if this is always like that. This is why I just extract the first element.